src/cascadia/TerminalApp/TabHeaderControl.xaml

```diff
@@ -11,6 +11,14 @@ the MIT License. See LICENSE in the project root for license information. -->
 
     <StackPanel x:Name="HeaderStackPanel"
                 Orientation="Horizontal">
+        <ProgressRing x:Name="HeaderProgressRing"
+                      IsActive="{x:Bind IsProgressRingActive, Mode=OneWay}"
+                      Visibility="{x:Bind IsProgressRingActive, Mode=OneWay}"
+                      MinHeight="0"
+                      MinWidth="0"
+                      Height="15"
+                      Width="15"
+                      Margin="-7.5,0,8,0"/>
```

Comment:
We should definitely add a comment calling out this _insane_ margin, explaining that the icon will always have space reserved and what's going on here.